jinahub/encoders/image/ImageTorchEncoder/README.md

```diff
@@ -2,24 +2,57 @@
 
 **ImageTorchEncoder** wraps the models from [torchvision](https://pytorch.org/vision/stable/index.html).
 
-**ImageTorchEncoder** encodes `Document` blobs of type a `ndarray` and shape Batch x Height x Width x Channel 
-into a `ndarray` of Batch x Dim and stores them in the `embedding` attribute of the `Document`.
+**ImageTorchEncoder** encodes `Document` blobs of type `ndarray` and shape Height x Width x Channel 
+into an `ndarray` of shape `embedding_dim` and stores them in the `embedding` attribute of the `Document`.
 
 
+## Usage
 
 
-## Usage
+Use the prebuilt images from Jina Hub in your Python codes and add it to your Flow:
+```python
+from jina import Flow
+
+flow = Flow().add(uses='jinahub+docker://ImageTorchEncoder')
+```
+### Encoding example
```

Comment:
I think it would make sense to show example of how an image (not just a numpy array) is encoded

Reply:
done